sqlobject/src/main/java/org/jdbi/v3/sqlobject/SqlObjectFactory.java

```diff
@@ -43,11 +41,24 @@
 public enum SqlObjectFactory implements ExtensionFactory<SqlObjectConfig> {
     INSTANCE;
 
-    private static final MethodInterceptor NO_OP = (proxy, method, args, methodProxy) -> null;
+    private static final Object[] NO_ARGS = new Object[0];
+    private static final Method EQUALS_METHOD;
+    private static final Method HASHCODE_METHOD;
+    private static final Method TOSTRING_METHOD;
+
+    static {
+        try {
+            EQUALS_METHOD = Object.class.getMethod("equals", Object.class);
+            HASHCODE_METHOD = Object.class.getMethod("hashCode");
+            TOSTRING_METHOD = Object.class.getMethod("toString");
+        }
+        catch (NoSuchMethodException wat) {
+            throw new IllegalStateException("JVM error", wat);
+        }
+    }
```

Comment:
Extensions created with Handle.attach will not use the on-demand wrapper.

We probably want some equivalent tests for attached SQL objects. If those pass without these handlers, I'm happy to get rid of them.